src/EditorFeatures/Core.Wpf/StringIndentation/StringIndentationAdornmentManager.cs

```diff
@@ -39,13 +39,22 @@ protected override void AddAdornmentsToAdornmentLayer_CallOnlyOnUIThread(Normali
 
             foreach (var changedSpan in changedSpanCollection)
             {
+                // Don't do any work for a raw string if it doesn't at least intersect the view span. If it does
+                // intersect the view, then we'll want to draw whichever is in view.
                 if (!viewLines.IntersectsBufferSpan(changedSpan))
                     continue;
 
                 var tagSpans = TagAggregator.GetTags(changedSpan);
                 foreach (var tagMappingSpan in tagSpans)
                 {
-                    if (!ShouldDrawTag(changedSpan, tagMappingSpan, out _))
+                    // Intentionally not calling TryGetViewLine like other adornment managers do.  That helper ensures
+                    // that the end point of the tag maps to a line that is in view.  However, for raw-string
+                    // indentation adornments it's fine for that point to be offscreen, and we still want to draw the
+                    // indentation line in that case.
```

Comment:
So the fact that `GetTextViewLineContainingBufferPosition` returns `null` if the line is not visible in the viewport is an actually expected behavior?

Reply:
Correct. It's a *view* line. They only exist for the visible lines. That's the concept of a view vs a buffer.